services/rds/src/main/java/org/finra/gatekeeper/services/db/connections/model/LambdaDTO.java

```diff
@@ -0,0 +1,34 @@
+package org.finra.gatekeeper.services.db.connections.model;
+
+public class LambdaDTO {
+    String gatekeeperPassword;
```

Comment:
After careful consideration, the lambdas should be responsible for pulling down the passwords. This field should be something that can help the lambda grab the password. From what I saw we are loading the instance password into the lambda as part of the payload, I would like to strictly limit this to the lambda function as to limit the number of hops the password is taking from service to service.